examples/mugen/retrieval/video_clip.py

```diff
@@ -6,41 +6,54 @@
 
 import torch
 from torch import nn
-from torchmultimodal.utils.common import get_current_device
-from transformers import DistilBertConfig, DistilBertModel, DistilBertTokenizer
+from transformers import DistilBertConfig, DistilBertModel
 
 
 class TextEncoder(nn.Module):
-    """Encode text to a fixed-length vector
+    """Encode tokenized text to a fixed-length vector
 
-    Taken from MUGEN's text encoder
+    Adapted from MUGEN's text encoder
         (https://github.com/mugen-org/MUGEN_baseline/blob/main/lib/models/videoclip/modules.py)
+
+    Args:
+        pretrained (bool): whether to use a pretrained model or not
+        model_name (str): name of pretrained model, used when pretrained is True
+        model_config (dict): model config for DistilBERT, used when pretrained is False
+        padding_value (int): value that was used to pad the input text
+
+    Inputs:
+        input_ids (Tensor): tensor of (batch, text_length) tokenized text
+
+    Returns:
+        Tensor: encoded text
```

Comment:
Add dimension here too? Would also elaborate on the purpose of indexing the last hidden states with the `CLS token` even though it's probably a well known practice for BERT model.